include/pmacc/identifier/value_identifier.hpp

```diff
@@ -49,19 +49,23 @@
  * @{
  */
 
-/** @param ... must be a device function/lambda with arguments worker and  IdGenerator
+/** @param ... Must be a device function/lambda with arguments worker, IdGenerator and source particle.
+ *             @attention source particle can be invalid, it is required to check the state via particles
+ *             isHandleValid() method.
```

Comment:
For my understanding, using `isHandleValid()` is not strictly required in the functor correct? There may be attributes which dont need the particle to be valid.

Reply:
Yes it's optional.

I will change it to `..., isHandleValid() can be used to check the status.`